stylize/__main__.py

```diff
@@ -57,6 +57,9 @@ def main():
 
     # register any formatter-specific arguments
     for formatter in formatters:
+        if formatter.get_command() == "false":
+            print("[ERR] A required dependency was not found. Check to see if clang-format is available on your path.")
```

Comment:
Hmm, that sounds like a good idea, I'll make the default return an empty string then. 
I'm going to squash all these commits once we finalize things btw...